wled00/wled20_ir.ino

```diff
@@ -1,8 +1,8 @@
 /*
- * Infrared sensor support for generic 24 key RGB remote
+ * Infrared sensor support for generic 24/40/44 key RGB remotes
  */
 
-#if defined(WLED_DISABLE_INFRARED) || defined(ARDUINO_ARCH_ESP32)
+#if defined(WLED_DISABLE_INFRARED)
```

Comment:
enable ESP32 as well as it is supported since v2.6.5 of the IRremoteESp8266 library.